test/sql/date/test_timestamp.cpp

```diff
@@ -134,3 +134,40 @@ TEST_CASE("Test storage for timestamp type", "[timestamp]") {
 	}
 	DeleteDatabase(storage_database);
 }
+
+TEST_CASE("Test timestamp functions", "[timestamp]") {
```

Comment:
Could you add some more tests with:
(1) NULL values as BOTH arguments
(2) Timestamps coming as input from a table, and interacting with either other columns or constants
-> e.g. like this:
```sql
SELECT AGE(t, TIMESTAMP '2014-04-17') FROM times;
SELECT AGE(TIMESTAMP '2014-04-17', t) FROM times;
SELECT AGE(t1, t2) FROM times;
```
(3) Also add a test with a selection vector, e.g.:
```sql
SELECT AGE(t1, t2) FROM times WHERE id > 2;
```

Reply:
I added all of the required tests. On Postgres, the NULL argument does not fail but it gives an empty result. On DuckDB we get Unsupported input types for function age(NULL, NULL). Is this the correct approach?

Reply:
AGE(NULL, NULL) should work as well, but this should be handled in the future by generic function overloading (which we don't have yet). For now it is fine that is throws an unsupported exception instead.

Reply:
Oh just to clarify, with NULL values as both arguments I meant all permutations of NULL and an argument, i.e.

```sql
SELECT AGE(t, NULL);
SELECT AGE(NULL, t);
SELECT AGE(NULL, NULL);
```

Also make these tests using values inside a table, and put a NULL as one of the values in the table.